modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -325,6 +325,10 @@ example-memcached    4         4         4            4           5m
 . *Clean up the resources:*
 +
 ----
+$ kubectl delete -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
```

Comment:
@bergerhoffer Sorry, my mistake. We should delete the `deploy/crds/cache_v1alpha1_memcached_crd.yaml` after deleting the `deploy/crds/cache_v1alpha1_memcached_cr.yaml`.

Reply:
No problem, I've moved deleting `deploy/crds/cache_v1alpha1_memcached_crd.yaml` to below deleting `deploy/crds/cache_v1alpha1_memcached_cr.yaml`. Let me know if this looks okay now:

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1684476/applications/operator_sdk/osdk-getting-started.html#building-memcached-operator-using-osdk_osdk-getting-started